MapboxMobileEvents/MMEEventsManager.m

```diff
@@ -323,7 +323,8 @@ - (void)sendTurnstileEvent {
                                                MMEEventSDKVersion: self.apiClient.hostSDKVersion,
                                                MMEEventKeyEnabledTelemetry: @([self isEnabled]),
                                                MMEEventKeyLocationEnabled: @([CLLocationManager locationServicesEnabled]),
-                                               MMEEventKeyLocationAuthorization: [CLLocationManager mme_authorizationStatusString]
+                                               MMEEventKeyLocationAuthorization: [CLLocationManager mme_authorizationStatusString],
+                                               MMEEventKeySkuID: self.skuID ?: [NSNull null]
```

Comment:
I had to look up `?:` as an operator. Elvis has left the building, and want's to unwrap an optional…